### ISSUE TITLE
Default styling for react

### ISSUE THREAD

**Issue by AWR14** [souporserious/react-media-player#56]
Does this come with any default styling?

**Comment by souporserious** [comment]
No, there aren't any styles that ship by default. It is unopinionated as far as styling goes. The CSS styles in the [example](https://github.com/souporserious/react-media-player/blob/master/example/main.scss) folder might help if you're looking for a place to start.
